tests/comparisons/Diff/default/the_diff_default_mysql.php

```diff
@@ -201,6 +201,7 @@ public function down(): void
         $this->table('articles')
             ->addColumn('content', 'text', [
                 'after' => 'rating',
+                'collation' => 'utf8_general_ci',
```

Comment:
On the CI the default collation is set to utf8mb4_general_ci. 

`sudo service mysql start mysql -h 127.0.0.1 -u root -proot -e 'CREATE DATABASE cakephp_test DEFAULT COLLATE=utf8mb4_general_ci;`

So that is why I had to add this wherever the test has a different collation.